src/store/slices/solanaSlice.ts

```diff
@@ -170,33 +170,38 @@ export const makeCollectablePayment = createAsyncThunk(
   ) => {
     if (!account?.solanaAddress) throw new Error('No solana account found')
 
-    const transfer = collectable.compression.compressed
-      ? await solUtils.transferCompressedCollectable(
-          cluster,
-          account.solanaAddress,
-          account.address,
-          collectable,
-          payee,
-        )
-      : await solUtils.transferCollectable(
+    try {
+      const transfer = collectable.compression.compressed
+        ? await solUtils.transferCompressedCollectable(
+            cluster,
+            account.solanaAddress,
+            account.address,
+            collectable,
+            payee,
+          )
+        : await solUtils.transferCollectable(
+            cluster,
+            account.solanaAddress,
+            account.address,
+            collectable,
+            payee,
+          )
+
+      // If the transfer is successful, we need to update the collectables
+      if (!transfer.txn?.meta?.err) {
+        dispatch(fetchCollectables({ account, cluster }))
+      }
+
+      return await dispatch(
```

Comment:
havent used this return await syntx. It just ensure the catch block will catch errors?

Reply:
Hmm no actually don't need this. Will remove

Reply:
Actually nvm now I remember that postPayment was throwing errors so I added the await lol